gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
OK I defer to @jeffposnick on this one. I thought it was for server-side templating, which we do do. My understanding was that sw-precache creates a hash for each each resource to determine when it changes.